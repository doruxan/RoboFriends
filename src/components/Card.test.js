import { shallow } from 'enzyme'
import React from 'react'
import Card from './Card'


it('is rendering a card component', () => {
    expect(shallow(<Card item={{name:'doruk', email:'doruxan@gmail.com'}} />)).toMatchSnapshot()
})