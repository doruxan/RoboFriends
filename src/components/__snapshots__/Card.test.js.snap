// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`is rendering a card component 1`] = `
<div
  className="tc bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5"
>
  <img
    alt="RobotImg"
    src="https://robohash.org/doruk?size=s200x200"
  />
  <div>
    <h2>
      doruk
    </h2>
    <p>
      doruxan@gmail.com
    </p>
  </div>
</div>
`;
